src/services/api.ts

```diff
@@ -120,6 +121,17 @@ export async function getUsers(ids: number[] = []): Promise<UserDTO[]> {
 	return getJSON('/users', { ids: ids.join(',') });
 }
 
+export async function manageAPIKey(action: string): Promise<void> {
```

Comment:
I don't like magic string in the argument. @pyoner maybe enum/union of strings/symbols is better?

Reply:
@knopki I think union of string literal is enough
enum generate additional code, [see](https://www.typescriptlang.org/play?#code/PTAEFUDsEsHtIMoBcBO1IHMAy0kFMUBDAG1AFgAoJATwAc8IZ5k1Md8jSBeUAcgDNYsXqAA+fAEaEUI8bykAvXgG5KlEKACikAK4BbUIcp5dB7ftABvSodAAxAPIPQPAUN4AaG4YBCAQQAlF0lpT29QfwAtYPlCJUoAXzUKDQBheABnJHMDIwoAY0ykUBMLdMgsnKtwx2dXQWEvClt-INcpGSaWv2j2uN5E5MoAN2lQQgAuRjhEVHRsXAISYIAiDpWRsYkpqp4cgDpWzZRQfKnyytNgi+zTQ8DkoA)